### PR TITLE
DOCS-4569 Updated body scanning note SID Phase II

### PR DESCRIPTION
**Pull request by valkyrie69**
Updated outdated note about disabling body scanning

### CODE REVIEW

File: modules/ROOT/pages/create-waf-policy.adoc [hunk 1]
```diff
@@ -25,12 +25,14 @@ When you hover over the `i` to the right of each rule, the rule ID range for tha
 image::waf-rule-set.png[]
 +
 . In *Advanced performance options*, select from the following options: +
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/create-waf-policy.adoc]: You can only select one of the options, so I'd put "select one of the following options:" until a third, non-exclusionary option is added.

File: modules/ROOT/pages/create-waf-policy.adoc [hunk 2]
```diff
@@ -25,12 +25,14 @@ When you hover over the `i` to the right of each rule, the rule ID range for tha
...
-* *Disable body scanner*. If you do not check this option, the request body is scanned.
-+
+* *Disable body scanner*. By default, the request body is scanned unless the body is larger than 1 MB, in which case the scan is skipped. +
+Check this option if request body scanning isn't needed, or to reduce CPU consumption. +
+
 [NOTE]
 If a body is larger than 1 MB, the scan is skipped, even if you do not select *Disable body scanner*. 
-* *Detect sensitive information* - You can select this option only if the *Disable body scanner* option is not selected. +
-If you select this option, the request body is scanned for sensitive information, and when you apply the WAF policy to Runtime Fabric, the log summary (`rtfWafSecurityPolicySummary`) records the count and number (rule ID) of the sensitive information pattern that is detected. +
+* *Detect sensitive information* - If you select this option, the request body is scanned for sensitive information, and when you apply the WAF policy to Runtime Fabric, the log summary (`rtfWafSecurityPolicySummary`) records the count and number (rule ID) of the sensitive information pattern that is detected. +
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/create-waf-policy.adoc:33]: This option doesn't show up in the GitHub "View file" view. If it shows up in a local build, ignore this comment.

File: modules/ROOT/pages/create-waf-policy.adoc [hunk 3]
```diff
@@ -44,19 +46,22 @@ All the rulesets are disabled by default. For each rule, you can: +
...
-* *Disable body scanner* - This option works the same as it does for the request ruleset, however, there are some additional important caveats:
+* *Disable body scanner* - By default, the response body is scanned unless the body is larger than 1 MB, in which case the scan is skipped. +
+Check this option if response body scanning isn't needed, or to reduce CPU consumption. +
+This option works the same as it does for the request ruleset with some additional important caveats:
 +
 [IMPORTANT]
-If a body is larger than 1 MB, the body scan is skipped, even if you selected *Disable body scanner*. 
+If a body is larger than 1 MB, the body scan is skipped, even if you do not select *Disable body scanner*. 
 +
 * Response messages must have one of the following MIME types in the content type header, or the WAF rules will not be evaluated:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/create-waf-policy.adoc:53]: In GitHub View file, this shows up at the same level as the bullets, making it look like a third option. Not clear what it goes with.

File: modules/ROOT/pages/create-waf-policy.adoc (hunk 1 above)
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/create-waf-policy.adoc]: You can only choose one option, so I'd say "select one of the following options" here and in the same section below.

**Review by Mulesoft-Mysti — APPROVED** [review]
Great rewrite, makes perfect sense! Just some non-show-stopping suggestions.

File: modules/ROOT/pages/sid-reference.adoc [hunk 4]
```diff
@@ -89,34 +91,74 @@ This is an example of the log, which shows one violation was detected in the col
...
+
+== Rule Categories
+
+Sensitive information detection rules belong to rule categories, which include: 
+
+* PII: Personally identifiable information, such as date of birth, social security number, passport number, and so on. 
+* PCI: Payment card information, such as a Visa or Mastercard number. 
+* PHI: Protected health information.
 
 == View Rule Categories and IDs in RAML
```
Comment (fumingshih) [inline modules/ROOT/pages/sid-reference.adoc]: @valkyrie69, I checked the RAML file, there's no info for "SID categories". I think we should change the title to 
"View Rule Name and Rule Id" 
Here's an example of the content in the RAML file. 
`        
{
          "id": 600780,
          "ver": "",
          "rev": "1",
          "phase": "response",
          "severity": "WARNING",
          "accuracy": "5",
          "maturity": "1",
          "taglist": [
            "MULESOFT_SID"
          ],
          "msg": "VISA CARD",
          "variables": "RESPONSE_BODY",
          "operator": "UkVTUE9OU0VfQk9EWSAoPzwhWy5cdy1dKTRbMC05XXszfShbIC1dPylbMC05XXs0fVwxWzAtOV17NH1cMVswLTldezR9KD8hWy5cdy1dKQo=",
          "paranoia": 1,
          "chain": "no"
        },
`

File: modules/ROOT/pages/sid-reference.adoc [hunk 5]
```diff
@@ -89,34 +91,74 @@ This is an example of the log, which shows one violation was detected in the col
...
 
-You can view the the SID categories and rule IDs in the Anypoint Security RAML (`security-fabric-policies-api-<version>.raml`). 
+You can view the the SID categories and rule IDs in the Anypoint Security Policies RAML. 
+
+. Go to the public https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[MuleSoft developer portal], and search for "Anypoint Security Policies API".
+. Download the RAML and extract the ZIP files.
+. Navigate to `<Download_location>/anypoint-security-policies-api-<version>-raml/dataTypes/policies/WafRules/Rulesets.json`.
+
```
Comment (fumingshih) [inline modules/ROOT/pages/sid-reference.adoc:180]: The Rulesets.json file actually includes WAF rulesets other than SID rules. We should specify how to find SID rules.
Should add step 4:

4. Search keyword "MULESOFT_SID" which will show the SID rule and its details, including name (msg) and id.

**Review by fumingshih — APPROVED** [review]

**Pull request merged**